hack/internal/test-images/ray/mock_server.py

```diff
@@ -0,0 +1,243 @@
+import os
```

Comment:
Is this code copy-pasted from somewhere? If so please add the info in a comment, and README.md